templates/zerver/integrations/errbot.md

```diff
@@ -1,18 +1,20 @@
+# Zulip Errbot integration
+
 Run your favorite chatbot in Zulip!
 
-1. [Install errbot](https://errbot.readthedocs.io/en/latest/user_guide/setup.html)
-   and follow to instructions to set up a `config.py`.
+{start_tabs}
 
-1. Check out our [Errbot integration package for Zulip](https://github.com/zulip/errbot-backend-zulip).
-   Clone this repository somewhere convenient.
+1. [Install errbot][install-errbot], and follow the instructions to set up a
+   `config.py`.
 
-1. Install the requirements listed in `errbot-backend-zulip/requirements.txt`.
+1. Clone the [Errbot integration package for Zulip][errbot-package]
+   repository somewhere convenient, and install the requirements listed in
+   `errbot-backend-zulip/requirements.txt`.
 
 1. {!create-a-generic-bot.md!}
 
-1. Download your Zulip bot's `zuliprc` config file. You will need its content for the next step.
```

Comment:
Removed this instruction, and all mentions of `zuliprc` since the exact file is not needed anywhere, just the bot's info.
Instead added a tip with the link to the bot's info.